include/openssl/ssl.h

```diff
@@ -282,6 +282,9 @@ typedef int (*SSL_custom_ext_parse_cb_ex) (SSL *s, unsigned int ext_type,
 /* Typedef for verification callback */
 typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
 
+/* In TLSv1.3 allow a non-(ec)dhe based kex_mode */
+# define SSL_OP_ALLOW_NO_DHE_KEX                         0x00000001U
```

Comment:
Good catch. I've updated to use a new value that isn't caught by the 1.1.0 definition of SSL_OP_ALL. (Side note: Should SSL_OP_ALL have been changed???)